apps/builddao/widget/page/notifications.jsx

```diff
@@ -0,0 +1,63 @@
+const accountId = context.accountId;
+
+if (!accountId) {
+  return "Sign in with NEAR Wallet";
+}
```

Comment:
should we show a login button to navigate them to `join` page?
Like we do for feed I guess

Reply:
I think this sounds like a nice addition